debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -292,6 +292,11 @@ private boolean schemaChanged(List<ReplicationMessage.Column> columns, Table tab
                         return true;
                     }
                 }
+                PostgresType pgType = message.getType();
+                if (pgType.isEnumType() && !pgType.getEnumValues().contains(message.asString())) {
+                    logger.info("detected new enum value '{}' for column '{}'", message.asString(), columnName);
+                    return true;
+                }
```

Comment:
@jpechane I've got a few questions if you don't mind:
* Is this about how you would about solving this?
* How can I get the value from message so that I may check for its presence in the enum value list? `message.asString()` does not seem to be working out for me.
* It looks like `pgType.getEnumValues()` returns a list of strings. Does this mean that only enums of strings are currently supported?

Reply:
I'll leave it to @jpechane to answer in more detail.

> Does this mean that only enums of strings are currently supported?

Yes, PG enums are enums of strings. In general, the suggested approach isn't 100% water-proof: messages would be emitted with the old schema until the first message using the new value arrives; also, if an enum constant gets removed, the schema would never be updated. But I suppose it's as good as it gets.

Reply:
@bradengroom Hi, I am sorry for letting the PR to stale. So this is eactly how it should look like.
Please take care of the compilation errors and we are goog to go as the test seems to be spot on.
Regarding string type - does POstgreSQL suport non-integer enums?

Reply:
No, there are only String enums possible.